lib/Byng/Pimcore/Sitemap/Generator/SitemapGenerator.php

```diff
@@ -107,9 +149,22 @@ private function addUrlChild(Document $document)
             $document instanceof Document\Page &&
             !$document->getProperty("sitemap_exclude")
         ) {
-            echo $this->hostUrl . $document->getFullPath() . "\n";
+            $fullPath = $document->getFullPath();
+
+            // Remove the site path (if any) from the full path
+            $rootPath = (string)$this->host->rootPath;
+            if (!empty($rootPath) && strpos($fullPath, '/' . $rootPath) === 0) {
+                if ($fullPath === '/' . $rootPath) {
+                    // Special case for the homepage
+                    $fullPath = '';
+                } else {
+                    $fullPath = str_replace('/' . $rootPath . '/', '/', $fullPath);
+                }
+            }
+
+            echo $this->hostUrl . $fullPath . "\n";
```

Comment:
I don't think we need it, but it was [in the original class](https://github.com/byng-systems/pimcore-sitemap-plugin/blob/master/lib/Byng/Pimcore/Sitemap/Generator/SitemapGenerator.php#L110), I just kept it.